### PR TITLE
[OSD-12252] disableAllDefaultSources for FedRamp 

### PR DESCRIPTION
**Pull request by theautoroboto**
### What type of PR is this?
_(bug/feature/cleanup/documentation)_
workaround

### What this PR does / why we need it?
We are running into a bug in FedRAMP and need to disableAllDefaultSources for operator sources.  Once the issues are addressed we can remove this change.  This is being cause by an issue with pull secrets in FedRamp specifically

### Which Jira/Github issue(s) this PR fixes?
[12252](https://issues.redhat.com/browse/OSD-12252)

_Fixes #_

### Special notes for your reviewer:
This is only a temporary fix

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: deploy/osd-curated-operatorsources-revert/fedramp/00-operatorhub.cr.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: config.openshift.io/v1
```
Comment (cblecker) [inline deploy/osd-curated-operatorsources-revert/fedramp/00-operatorhub.cr.yaml:1]: Please place these files in a separate directory under deploy. Something like `deploy/fedramp-operatorhub/` or the like. Don't want this getting mixed into the curated operatorsources stuff (it's really legacy)

File: deploy/osd-curated-operatorsources-revert/fedramp/10-osd-patch-subscription-source.CronJob.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+apiVersion: batch/v1beta1
```
Comment (cblecker) [inline deploy/osd-curated-operatorsources-revert/fedramp/10-osd-patch-subscription-source.CronJob.yaml:1]: Don't need the clusterrole/binding or cronjob. They're legacy.

**Review by cblecker — CHANGES_REQUESTED** [review]
/hold

File: deploy/osd-curated-operatorsources-revert/fedramp/10-osd-patch-subscription-source.CronJob.yaml (hunk 2 above)
Comment (theautoroboto) [reply deploy/osd-curated-operatorsources-revert/fedramp/10-osd-patch-subscription-source.CronJob.yaml]: 3 files removed

File: deploy/osd-curated-operatorsources-revert/fedramp/00-operatorhub.cr.yaml (hunk 1 above)
Comment (theautoroboto) [reply deploy/osd-curated-operatorsources-revert/fedramp/00-operatorhub.cr.yaml]: moved to its own folder

File: deploy/fedramp-operatorhub/00-osd-patch-subscription-source.ServiceAccount.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (cblecker) [inline deploy/fedramp-operatorhub/00-osd-patch-subscription-source.ServiceAccount.yaml:1]: Oop sorry, this one can get removed too

Comment (theautoroboto) [reply deploy/fedramp-operatorhub/00-osd-patch-subscription-source.ServiceAccount.yaml]: Removed

**Review by cblecker — APPROVED** [review]
/lgtm
/approve
/label tide/merge-method-squash

**Pull request merged**